modules/programs/direnv.nix

```diff
@@ -93,9 +92,11 @@ in {
     };
 
     xdg.configFile."direnv/direnvrc" = let
+      package =
+        pkgs.nix-direnv.override { inherit (cfg.nix-direnv) enableFlakes; };
```

Comment:
Since we never removed this flag, this should be save to merge any time and will become only effective when https://github.com/NixOS/nixpkgs/pull/147998 is in the release channel.